MultimodalQnA/docker_compose/intel/cpu/xeon/README.md

```diff
@@ -83,8 +83,6 @@ export WHISPER_PORT=7066
 export WHISPER_SERVER_ENDPOINT="http://${host_ip}:${WHISPER_PORT}/v1/asr"
 export WHISPER_MODEL="base"
 export MAX_IMAGES=1
-export ASR_ENDPOINT=http://$host_ip:$WHISPER_PORT
-export ASR_PORT=9099
 export ASR_SERVICE_PORT=3001
 export ASR_SERVICE_ENDPOINT="http://${host_ip}:${ASR_SERVICE_PORT}/v1/audio/transcriptions"
 export REDIS_DB_PORT=6379
```

Comment:
On line 165 and 271, could you changes those headers from ASR to Whisper Server or something?